### ISSUE TITLE
Can I use this library for video streaming 

### ISSUE THREAD

**Issue by peererror** [lsalzman/enet#81]
I am developing an application that stream archive files from user file system .Can use this library for this purpose .the video file size ~500mb

**Comment by inlife** [comment]
I believe it's better to use TCP based solutions for that matter

**Comment by Croydon** [comment]
> I believe it's better to use TCP based solutions for that matter

Yes, if you want to transfer files like that, you want to make sure, that all bits of the files gets transferred reliable. Enet is not made for that.

The question was answered and this issue can probably be closed.
